.github/workflows/run-acceptance-tests.yml

```diff
@@ -22,9 +22,8 @@ env:
   SIGNING_KEY_ID: ${{ secrets.JAVA_SIGNING_KEY_ID }}
   SIGNING_PASSWORD: ${{ secrets.JAVA_SIGNING_PASSWORD }}
   SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
-  TRAVIS_OS_NAME: linux
   TF_APPEND_USER_AGENT: pulumi
-  VERSION_PREFIX: 4.0.0
```

Comment:
I'm wondering if removal of VERsION_PREFIX by ci-mgmt is actually what broke the build.